seed/index.js

```diff
@@ -88,6 +88,7 @@ Observable.combineLatest(
             challenge.suborder = index + 1;
             challenge.block = dasherize(blockName);
             challenge.blockId = block.id;
+            challenge.blockName = blockName;
```

Comment:
Like I said, there is no reason to modify the seed files. This information is already in the challenges/blocks/redux store.

It may seem overly complicated but making sure that there are not multiple source of information is important.

Redux provides the flexibility to store the minimum required and selectors allow you to build up derived data and colocate.

Think of the entities as the DB and selectors as the query.